### PR TITLE
LPD-19284 Update LockedKBArticleModal JS tests

### PR DESCRIPTION
**Pull request by marcogalluzzi**
https://liferay.atlassian.net/browse/LPD-19284

### CODE REVIEW

File: modules/apps/knowledge-base/knowledge-base-web/test/admin/js/components/LockedKBArticleModal.js [hunk 1]
```diff
@@ -39,8 +41,24 @@ describe('LockedKBArticleModal', () => {
 		expect(queryByText('article-in-edition')).not.toBeInTheDocument();
 	});
 
-	it('renders the modal when try to edit/expire/delete a locked article', () => {
-		const {getByText} = render(<LockedKBArticleModal open={true} />);
+	it('does not render the modal first as group admin', () => {
```
Comment (ambrinchaudhary) [inline modules/apps/knowledge-base/knowledge-base-web/test/admin/js/components/LockedKBArticleModal.js:44]: I think this test is unneccesary. While _open={false}_ nothing will be rendered, doesn't matter the `groupAdmin` value. And this is already covered by the first test.
And when both parameters are `true`, you are already testing `queryByText('article-in-edition-by-user-x-description')` in your next test **'renders the modal as group admin when try to edit/expire/delete a locked article'**

File: modules/apps/knowledge-base/knowledge-base-web/test/admin/js/components/LockedKBArticleModal.js [hunk 2]
```diff
@@ -69,5 +114,39 @@ describe('LockedKBArticleModal', () => {
 		});
 
 		expect(getByText('article-in-edition')).toBeInTheDocument();
+		expect(getByText('ok')).toBeInTheDocument();
+	});
+
+	it('renders the modal as group admin when try to move a locked article', async () => {
```
Comment (ambrinchaudhary) [inline modules/apps/knowledge-base/knowledge-base-web/test/admin/js/components/LockedKBArticleModal.js:120]: This test also covers the previous one (what we want to test is that a modal is opened on the move action - from js- and it is enough to test it as group admin). So I would suggest to delete the previous test `'renders the modal as non group admin when try to move a locked article'.`

**Review by ambrinchaudhary — CHANGES_REQUESTED** [review]
This are the results of coverage when the 2 tests that I suggest are removed (and the result is the same with those tests, which confirms they are not necessary), which is a very good result. 

Screenshot 2024-03-22 at 12 45 38

Once removed, this pull can be forwarded. 
Good work here @marcogalluzzi

**Comment by ambrinchaudhary** [comment]
Also, @marcogalluzzi, I don't get the result of the coverage if I run `npm run test --coverage`. So it seems that it only work with` yarn test --coverage`. Right now I can only suggest you to install `yarn`, but... it is also true that you won't need to do frontend unit test regularly, so it is up to you ;-)

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by marcogalluzzi** [comment]
ci:test:stable

**Comment by marcogalluzzi** [comment]
ci:test:stable

**Comment by adolfopa** [comment]
ci:forward:force

**Pull request closed without merging**